components/date-time/time.js

```diff
@@ -122,43 +182,103 @@ class TimePicker extends Component {
 
 	render() {
 		const { is12Hour } = this.props;
-		const { minutes, hours, am } = this.state;
+		const { day, month, year, minutes, hours, am } = this.state;
 
 		return (
 			<div className="components-time-picker">
-				<input
-					className="components-time-picker__input"
-					type="text"
-					value={ hours }
-					onChange={ this.onChangeHours }
-					onBlur={ this.updateHours }
-				/>
-				<span className="components-time-picker__separator">:</span>
-				<input
-					className="components-time-picker__input"
-					type="text"
-					value={ minutes }
-					onChange={ this.onChangeMinutes }
-					onBlur={ this.updateMinutes }
-				/>
-				{ is12Hour && <div>
-					<Button
-						isDefault
-						className="components-time-picker__am-button"
-						isToggled={ am === 'AM' }
-						onClick={ this.updateAmPm( 'AM' ) }
-					>
-						{ __( 'AM' ) }
-					</Button>
-					<Button
-						isDefault
-						className="components-time-picker__pm-button"
-						isToggled={ am === 'PM' }
-						onClick={ this.updateAmPm( 'PM' ) }
-					>
-						{ __( 'PM' ) }
-					</Button>
-				</div> }
+				<fieldset key="date-time-form" className="date-time-form">
```

Comment:
fieldset should use a `<legend>` element to give the group a name

Reply:
I aggree!